src/content/docs/en/guides/integrations-guide/db.mdx

```diff
@@ -92,6 +92,31 @@ export default defineDb({
 })
 ```
 
+## Configuration
+
+### `mode`
+
```

Comment:
```suggestion

<p>

**Type:** `'node' | 'web'`<br />
**Default:** `'node'`<br />
<Since v="0.18.0" />
</p>

```
Make this accurate, but we need the api reference stuff here!